src/pages/CreateEvent.js

```diff
@@ -0,0 +1,118 @@
+// src/components/ProfileForm.js
+import React, { useState } from "react";
+import "./CreateEvent.css"; // Import the CSS file
+
+const locations = ["Copenhagen", "Aarhus", "Odense", "Aalborg"];
+const petTypes = ["Dog", "Cat", "Bird", "Other"];
+
+const ProfileForm = () => {
+  const [location, setLocation] = useState("");
```

Comment:
Consider introducing state management for form inputs to handle changes and submission logic